heron/scheduler-core/src/java/com/twitter/heron/scheduler/utils/SchedulerUtils.java

```diff
@@ -179,19 +179,37 @@ private SchedulerUtils() {
    *
    * @param config The static config
    * @param runtime The runtime config
-   * @param containerIndex the executor/container index
+   * @param shardId the executor/container index
    * @param ports a map of ports to use where the key indicate the port type and the
    * value is the port
    * @return String[] representing the command to start heron-executor
    */
   public static String[] getExecutorCommand(
       Config config,
       Config runtime,
-      int containerIndex,
+      int shardId,
```

Comment:
Can this be done better?

Reply:
The flag we set is called shard so i updated the variable name to match. We can probably do this better but that is probably a bigger change that we can address in another pr.